schemas/org.notes.gschema.xml

```diff
@@ -119,7 +119,6 @@
 			<summary>Stylesheet modifications for renderer to use</summary>
 			<description>This data is the actual CSS the renderer will use</description>
 		</key>
-	  	<child name="notebooks" schema="org.notes.notebook_data.notebook" />
```

Comment:
Right now that's legacy code from before i started using a Database. The notebook schema stored the stylesheet & stylesheet-changes, and since you can have "notebooks in notebooks", i used this method in order to store the values correctly. 

I will be removing it on the next "mayor" release, but just not right now since removing it could cause people to loose their data :wink: